src/main/java/com/oltpbenchmark/Results.java

```diff
@@ -84,6 +85,10 @@ public Histogram<TransactionType> getRetryDifferent() {
         return retryDifferent;
     }
 
+    public Histogram<TransactionType> getZeroRows() {
```

Comment:
Can we print number of zero_rows to the command line o/p just like rejected transactions? It will help while running workloads manually